src/store/page/page_element.ts

```diff
@@ -81,7 +81,7 @@ export class PageElement {
 		const element = new PageElement(pattern, false, parent);
 
 		if (!pattern) {
-			console.warn(`Ignoring unknown pattern "${patternPath}"`);
+			// console.warn(`Ignoring unknown pattern "${patternPath}"`);
```

Comment:
This one we should probably keep until we have a better logging

Reply:
I would say that in general the `console.warn`s should be keept and only the `console.log`s should be removed.
In all cases where there is a `console.warn` we have something unexpected what we probably also should show the user.

Reply:
Makes ssense.